boards/arc/hsdk4xd/doc/index.rst

```diff
@@ -0,0 +1,525 @@
+.. _hsdk4xd:
+
+DesignWare(R) ARC(R) HS4x/HS4xD Development Kit
+###############################################
+
+Overview
+********
+
+The ARC HS4x/HS4xD Development Kit is the next revision of Synopsys HSDK board
+(ref. :doc:`../../hsdk/doc/index.rst`). It includes a multicore ARC HS4xD-based chip
```

Comment:
Note than we can use [ancor ](https://github.com/zephyrproject-rtos/zephyr/blob/2dbccf065429836bc311a5deef1ba9782725a0f9/boards/arc/hsdk/doc/index.rst?plain=1#L1)here instead of path to file. I.e:

```suggestion
The ARC HS4x/HS4xD Development Kit is the next revision of :ref:`Synopsys HSDK board <hsdk>`.
It includes a multicore ARC HS4xD-based chip
```

Reply:
PS: Haven't checked the `:doc:` part - if it wotks with it or not.